src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -32,28 +32,22 @@
 
 DATA_SAVE_ID  = wx.NewId()
 
-
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var to exclude subplots
+        self.toolitems = tuple(el for el in NavToolbar.toolitems\
+                if el[0] != 'Subplots')
```

Comment:
Please define `toolitems` as class attribute **above** the `__init__` method.  Just drop `self.` and move it to line 37.

Also, do not use backslash continuation for an expression in parentheses.  The parentheses already imply line-continuation.  Make sure `if` gets aligned as per PEP8.